src/masternode-payments.cpp

```diff
@@ -777,16 +777,10 @@ std::string CMasternodePayments::ToString() const
     return info.str();
 }
 
-bool CMasternodePayments::IsEnoughData(int nMnCount) {
-    if(GetBlockCount() > nMnCount * nStorageCoeff && GetBlockCount() > nMinBlocksToStore)
```

Comment:
I though of it too. Imo this condition doesn't help too much because you can have semi-full or near-empty blocks and it will pass, it mostly vote count that matters.